bootstrap.php

```diff
@@ -947,7 +948,7 @@ function ($app) {
     function ($app) {
         return new \CultuurNet\UDB3\Role\ReadModel\Users\RoleUsersProjector(
             $app['role_users_read_repository'],
-            $app['user_identity_resolver']
+            $app[Auth0UserIdentityResolver::class]
```

Comment:
I think we can now also remove the `user_identity_resolver` definition itself , right?